nightmare/paths/graphs.spec.js

```diff
@@ -0,0 +1,146 @@
+/*eslint max-len: ["error", 500], */
```

Comment:
@elric-wamugu this is no needed, file has less lines than the default configured on eslint.